app/frontend/public/pos/login/styles.css

```diff
@@ -0,0 +1,19 @@
+body {
+    background-color: #333;
+}
+
+.container {
+    display: flex;
+    justify-content: center;
+    height: 100vh;
+}
+
+.card {
+    margin: auto;
+    padding: 32px;
+}
+
+input,
+button {
+    width: 400px;
+}
```

Comment:
We ending files without whitespace Jason? 🤨

Reply:
???